cve_bin_tool/parsers/python.py

```diff
@@ -33,8 +33,8 @@ def run_checker(self, filename):
             )
         except subprocess.CalledProcessError as e:
             self.logger.error(e.output)
-            pip_version = subprocess.check_output(["pip3", "--version"])
-            pip_version = float(pip_version[6:10])
+            pip_version = str(subprocess.check_output(["pip3", "--version"]))
```

Comment:
If we're in here anyhow, let's put a comment showing a "normal" output of `pip3 --version` so we know what we're parsing

```suggestion
            pip_version = str(subprocess.check_output(["pip3", "--version"]))
            # Output will look like: 
            # pip 20.0.2 from /usr/lib/python3/dist-packages/pip (python 3.8)
```

(yes, I'm doing this on a virtualenv that has an old version of pip... you might want to generate your own string.)